DESCRIPTION

```diff
@@ -38,7 +38,8 @@ Imports:
     httr (>= 1.0.0),
     curl,
     sp,
-    utils
+    utils,
+    progress
```

Comment:
maybe just go ahead and specify a minimum version of `progress` as the one you are using?